go.mod

```diff
@@ -0,0 +1,15 @@
+module smart-health-cards-go
+
+go 1.16
+
+require (
+	github.com/google/go-cmp v0.5.8 // indirect
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/lestrrat-go/jwx v1.2.25 // indirect
+	github.com/lestrrat-go/jwx/v2 v2.0.4
+	github.com/lestrrat/go-jwx v0.9.1 // indirect
+	github.com/skip2/go-qrcode v0.0.0-20200617195104-da1b6568686e
+	github.com/stretchr/testify v1.8.0 // indirect
+	golang.org/x/crypto v0.0.0-20220427172511-eb4f295cb31f // indirect
+	gopkg.in/square/go-jose.v2 v2.6.0
+)
```

Comment:
Can you try running `go mod tidy` to see if these dependencies are still up to date?